### DISCUSSION TITLE
Get  .app / .ipa file from expo project without paid apple account

### DISCUSSION THREAD

**Discussion by esrefbozkurt** [DevSkillsHQ/discusssions#62] [Help] [answered]
After setting up everything on Mac  and using the xcode simulator with npx expo start + i, I was able to run the simulator. Now that I have to upload an ipa-file, i tried to get it through expo build:ios which is now eas build -p ios. I tried but failed because I need an paid apple acount to get the file. How is the approach for this ? Please Help

**Comment by ludvig** [comment, marked as answer]
Hi @esrefbozkurt, very valid concerns, it is fine to not provide the ipa-file and simply include the screen recording only. We will look into improving this aspect of the specific challenge for the future. Thanks for the input!

**Reply by esrefbozkurt** [reply]
Hey @ludvig thanks for the answer. Alright, I added the screen recording to the readme file inside the project.
